src/cf-typography/src/atoms/links.less

```diff
@@ -109,8 +39,6 @@
 
         position: relative;
 
-        padding-right: 1.25em;
-
         &:after,
         &:before {
```

Comment:
Are you sure? This is for positioning the icon when it's in a jump link at small sizes, which is essentially a block link situation.